src/user-guide/writing-workflows/scheduling.rst

```diff
@@ -325,8 +325,22 @@ inferred from context (rules below).
    except where specific tasks, if any, depend on :term:`clock triggers <clock
    trigger>`.
 
+
+Format 3: ``R[limit?]/[datetime]/[interval]``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
It feels a bit awkward talking about format 3, then 4, then 1. But that's probably the most sensible order